hyp-content-for.js

```diff
@@ -1,13 +1,13 @@
 angular.module("hypContentFor", [])
-  .constant("HYP_CONTENT_FOR_IDS", [ ])
+  .value("HYP_CONTENT_FOR_IDS", { })
```

Comment:
Yeah, it kinda made more sense as the content of the collection would be changing. And the array thing was just wrong, it was supposed to be an object from the get-go. :smiley: